src/scipp/plotting/__init__.py

```diff
@@ -156,3 +157,11 @@ def plot(*args, **kwargs):
         plt.ion()
 
     return output
+
+
+def plot(*args, **kwargs):
+    if config.get('use_plopp'):
```

Comment:
```suggestion
    if config['plot'].get('use_plopp'):
```

`config` is not a dict, but `config['plot']` is. Though we should probably add a `get` method.